### DISCUSSION TITLE
Will Graphin support G6 version 5?

### DISCUSSION THREAD

**Discussion by copsman** [antvis/Graphin#519] [Q&A]
Hi,
i love using Graphin, and now i am wondering will Graphin support the new version of G6 Version 5 or not.

If it will be supported, any estimated time we expect an update?

thanks!

**Comment by koteckiokamil** [comment]
+1 same question

**Reply by Kogl1n** [reply]
Is this project dead?

**Comment by hustcc** [comment]
The latest version is based on G6 v5, but it is not compatible with older versions. It greatly simplifies the architecture, focuses on React integration, and introduces no additional concepts or configurations.

**Reply by hustcc** [reply]
Because it is simple enough, the code is relatively stable and sees little to no iteration. Similar to echarts-for-react, as long as echarts itself doesn’t undergo major changes, the code hardly ever needs to be updated.
